packages/storybook/stories/va-select.stories.jsx

```diff
@@ -77,7 +75,6 @@ const Template = ({
         value={value}
         required={required}
         error={error}
-        aria-live-region-text={ariaLiveRegionText}
```

Comment:
Is this related to date things?

Reply:
It was not in use and seems like we forgot to remove it.

Reply:
I will undo the change and write up a ticket for it instead